### PR TITLE
Lucy documentation! 

### PR DESCRIPTION
**Pull request by SeaZeeZee**
In the right place this time, with the linedrawing for the front page added.

### CODE REVIEW

File: designs/lucy/src/pocket.js [hunk 1]
```diff
@@ -51,10 +51,10 @@ export default function (part) {
       .line(points.middle)
       .attr('class', 'path fabric dashed')
 
-    points.logo = points.center.shiftOutwards(points.middle, pocketWidth / 5)
+    points.logo = points.centerLeft.shift(270, pocketWidth / 3)
```
Comment (BenJamesBen) [inline designs/lucy/src/pocket.js:54]: I note that the new location can sometimes cause the logo to encroach upon the left seam, for example if the width is decreased. Your current change is an improvement from what exists now so I'm okay with the change as is. However, you might want to consider lowering it a bit more.
Screen Shot 2022-08-22 at 3 22 12 PM

**Review by BenJamesBen — CHANGES_REQUESTED** [review]
You've accidentally included the `sites/lab/core.839` file as part of this PR, and it probably should be removed from the PR.

(I believe that if you delete the file from your GitHub SeaZeeZee:lucy-doc repository branch, it should remove itself from the PR automatically?)

Otherwise, the PR looks fine to me.

File: designs/lucy/src/pocket.js (hunk 1 above)
Comment (SeaZeeZee) [reply designs/lucy/src/pocket.js:54]: I will lower it! Skully hates me, but I want them on there!

**Review by joostdecock — CHANGES_REQUESTED** [review]
Please export your SVG from inkscape as a plain SVG. We have no use for all that inkscape-specific stuff.

**Comment by SeaZeeZee** [comment]
> Please export your SVG from inkscape as a plain SVG. We have no use for all that inkscape-specific stuff.

Done!

**Review by joostdecock — APPROVED** [review]

**Pull request merged**